pkg/types/installconfig.go

```diff
@@ -96,7 +96,10 @@ type Networking struct {
 	// Type is the network type to install
 	Type netopv1.NetworkType `json:"type"`
 
-	// ServiceCIDR is the ip block from which to assign service IPs
+	// MachineCIDR is the IP address space from which to assign machine IPs.
+	MachineCIDR ipnet.IPNet `json:"machineCIDR"`
```

Comment:
What about `MachineNetwork` to match the convention that we are moving towards https://github.com/openshift/api/blob/master/config/v1/types_network.go#L26-L40

Reply:
> What about `MachineNetwork` to match the convention...

Are you ok with me breaking convention with the other properties we have here in `Networking` now (which use `CIDR`)?  If we're going to make changes to the install-config, now would be the time to do that sort of thing.  Personally, I think the fact that these are properties of a `Networking` structure is enough namespacing, so if we don't feel a need to point out CIDR-ness, I'd be fine with just:

```go
type Networking struct {
  Clusters []netopv1.ClusterNetwork `json:"clusters,omitempty"`
  Machine ipnet.IPNet `json:"machine"`
  Service ipnet.IPNet `json:"service"`
  Type netopv1.NetworkType `json:"type"`
}
```